tritium-lib/src/main/java/com/palantir/tritium/proxy/Instrumentation.java

```diff
@@ -143,18 +144,18 @@ private Builder(Class<T> interfaceClass, U delegate) {
          * metric names are chosen based off of the interface name and invoked method.
          *
          * @param metricRegistry - TaggedMetricsRegistry used for this application.
-         * @param globalPrefix - Metrics name prefix to be used
+         * @param prefix - Metrics name prefix to be used
          * @return - InstrumentationBuilder
          */
-        public Builder<T, U> withTaggedMetrics(TaggedMetricRegistry metricRegistry, String globalPrefix) {
+        public Builder<T, U> withTaggedMetrics(TaggedMetricRegistry metricRegistry, String prefix) {
             checkNotNull(metricRegistry, "metricRegistry");
-            this.handlers.add(new TaggedMetricsServiceInvocationEventHandler(
-                    metricRegistry, globalPrefix));
+            String serviceName = (Strings.isNullOrEmpty(prefix)) ? interfaceClass.getName() : prefix;
```

Comment:
I've been favoring `.getSimpleName()` since it tends to be more representative of the service.

Reply:
I went with full name fallback default because its a bit more descriptive as the `service-name` will be simple name: `com.palantir.tritium.test.TestInterface:{service-name=TestInterface, endpoint=test}`